app/controllers/user_profiles_controller.rb

```diff
@@ -13,8 +13,8 @@ def edit
   end
 
   def new
-    @user_profile=UserProfile.new
-    @avatarsrc= "avatar/#{@user_profile.avatar_file_name}"
+    @user_profile = UserProfile.new
+    @avatarsrc = "avatar/#{@user_profile.avatar_file_name}"
```

Comment:
Sir if I will remove this line then small default image will not be shown in form